meshtastic/mt_config.py

```
@@ -8,32 +7,85 @@
+elif _state_keys != _annotated_state:
+    drift_message = (
+        "Drift between MODULE_STATE_DEFAULTS and type annotations — "
+        f"missing annotations: {_state_keys - _annotated_state}, "
+        f"missing defaults: {_annotated_state - _state_keys}"
+    )
+    if os.getenv("CI_ASSERT_MODULE_STATE_DRIFT", "").lower() in {
+        "1",
+        "true",
+        "yes",
+    }:
+        raise AssertionError(drift_message)  # noqa: TRY003
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (unused: `TRY003`)

Remove unused `noqa` directive

(RUF100)